awx/ui_next/src/components/LaunchPrompt/mergeExtraVars.js

```diff
@@ -0,0 +1,11 @@
+import yaml from 'js-yaml';
+
+export default function mergeExtraVars(extraVars, survey = {}) {
+  const vars = yaml.safeLoad(extraVars) || {};
+  return {
+    ...vars,
+    ...survey,
```

Comment:
do we not need to yaml.safeLoad the survey vars?

Reply:
Nope. The survey vars are an object built by Formik in `<SurveyStep>`, so they aren't a string like `extraVars` are (which came as a string from the server/codemirror)